core/eolearn/core/utils/parsing.py

```diff
@@ -250,7 +280,7 @@ def _validate_parsing_request(feature_type: FeatureType, name: Optional[str], eo
         if eopatch is not None and name is not None and (feature_type, name) not in eopatch:
             raise ValueError(f"Requested feature {(feature_type, name)} not part of eopatch.")
 
-    def get_feature_specifications(self) -> List[Tuple[FeatureType, object]]:
+    def get_feature_specifications(self) -> List[Tuple[FeatureType, Union[str, EllipsisType]]]:
```

Comment:
Why not use `Optional[str]` as well, like the list below is defined? Or does it not work that way? I understood the `...` be more used as input.

Reply:
Having `(FeatureType.DATA, None)` stand for all the data features was a workaround that we had to do before we handled the ellipsis type. And internally it's a bit cleaner. But we needed a method that returns them with `...` because of our filesystem module. Hence this method exists.